client/landing/gutenboarding/section.ts

```diff
@@ -0,0 +1,8 @@
+export const GUTENBOARDING_SECTION_DEFINITION = {
+	name: 'gutenboarding',
+	paths: [ '/gutenboarding' ],
+	module: 'gutenboarding',
+	secondary: false,
+	group: 'sites',
```

Comment:
Let's move to its own group:

```suggestion
	group: 'gutenboarding',
```

I'm not sure what all the implications are.

Reply:
> I'm not sure what all the implications are.

Probably not too big -- this is mostly about highlighting UI elements in the masterbar IIRC -- which is a non-entity for us.